pandas/core/frame.py

```diff
@@ -8394,6 +8394,124 @@ def isin(self, values):
                 self.columns,
             )
 
+    def value_counts(
+        self, normalize=False, sort=True, ascending=False, bins=None, dropna=True
+    ):
+        """
+        .. versionadded:: 1.0.0
+
+        Return a Series containing counts of unique rows in the DataFrame.
+
+        By default, rows that contain any NaN value are omitted from the
```

Comment:
Somewhere, maybe here, we sold note that the Series has a MultiIndex with one level per input column.